src/main/webapp/app/overview/course-card.component.html

```diff
@@ -1,102 +1,100 @@
-                    <div class="col-3 next-exercise-col">
-                        <h6 jhiTranslate="artemisApp.studentDashboard.cardExerciseLabel"></h6>
-                    </div>
-                    <div class="col-6 next-exercise-col">
-                        <fa-icon class="next-exercise-icon" [icon]="nextExerciseIcon" placement="right auto" [ngbTooltip]="nextExerciseTooltip | artemisTranslate" />
-                        <span class="next-exercise-title">{{ nextRelevantExercise.title }}</span>
-                    </div>
-                    <div class="col-3 next-exercise-col text-nowrap">
-                        @if (nextExerciseDueDate) {
-                            <div>
-                                <h6>{{ 'artemisApp.exercise.dueDate' | artemisTranslate }}:</h6>
-                                <h6>{{ nextExerciseDueDate | artemisTimeAgo }}</h6>
+        <div class="d-flex flex-column justify-content-between gap-1">
+            <div class="information-box-wrapper ms-1">
+                <div class="row mt-1">
+                    <div class="col-7" [ngClass]="{ 'col-12': !(exerciseCount && (totalReachableScore || totalAbsoluteScore)) }">
+                        @if (exerciseCount && (totalReachableScore || totalAbsoluteScore)) {
+                            <div class="align-self-center">
+                                <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardScore"></div>
+                                <div class="fw-bold">
+                                    <div
+                                        jhiTranslate="artemisApp.studentDashboard.points"
+                                        [translateValues]="{
+                                            totalAbsoluteScore: totalAbsoluteScore,
+                                            totalReachableScore: totalReachableScore,
+                                            totalRelativeScore: totalRelativeScore,
+                                        }"
+                                    ></div>
+                                </div>
+                            </div>
+                        } @else {
+                            <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardScore"></div>
+                            <div class="fw-bold">
+                                <span jhiTranslate="artemisApp.studentDashboard.noStatistics"></span>
+                            </div>
+                        }
+                        <hr />
+                        <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardExerciseLabel"></div>
+                        @if (nextRelevantExercise) {
+                            <a class="fw-bold btn-wrapper exercise-title exercise-guided-tour" [routerLink]="['/courses', course.id!, 'exercises', nextRelevantExercise.id!]">
+                                {{ nextRelevantExercise.title }}
+                            </a>
```

Comment:
I think for `course.id` this is a problem of the template in general, rather than this specific location - basically the entire template relies on `course.id` not being `undefined` (everything but the outermost div). Personally I'd be fine with leaving the assertion for `course.id` as is. No strong opinion either way